haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="https://github.com/grahams/alivejournal"
+COPYRIGHT="2000 Simon Huet
+	2010 Sean Graham"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitSrcRevision="11a4bed0a0800f66ce5069d3df6a0a3cc2be43aa"
+SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="198950a11a0cb38cd9093d08619991ad6bbd2d376e7b1999b09a8a556d9add68"
+SOURCE_FILENAME="alivejournal-$portVersion-$gitSrcRevision.tar.gz"
+SOURCE_DIR="alivejournal-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	alivejournal = $portVersion
+	app:AliveJournal = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir $docDir
+	install -t $appsDir objects/AliveJournal
+	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $docDir README
+	sed -n '/http://www.bebits.com/app/1920/q;p' $docDir/README
+	addResourcesToBinaries Resource.rdef $appsDir/AliveJournal
```

Comment:
Thanks @scottmc for the info about the RDEFS parameter in the Makefile.

@bach5000, this is good news. Now, to test a modified Makefile (i.e. with a "RDEFS = Resource.rdef"), you can create a patchset with the changes Scott explained, and see if everything builds find, and also check if the app has its icon.
(Don't forget to remove the call to addResourcesToBinaries in the recipe, otherwise you won't be sure that the resource is being inserted in the runtime by the very Makefile.)
Once you know that the modified Makefile works as expected, you can create a PR to change the "middle = 51" to "middle = 61" and the "RSRCS = Resource.rsrc" to "RDEFS = Resource.rdef".
The final step will be to drop the temporary patch you just created to test all this.